SWIG/scheduler.i

```diff
@@ -50,9 +50,16 @@ class Schedule {
     %rename("is-regular?") isRegular;
     #endif
   public:
-    Schedule(const std::vector<Date>&,
-             const Calendar& calendar,
-             BusinessDayConvention rollingConvention);
+	Schedule(const std::vector<Date>&,
+			 const Calendar& calendar = NullCalendar(),
+			 const BusinessDayConvention
+								convention = Unadjusted,
+			 boost::optional<BusinessDayConvention>
```

Comment:
This won't work at this time. We need to define typemaps for `boost::optional`, or to check if there are any available, otherwise SWIG won't know how to wrap it and will revert to an opaque class that we can't instantiate from Python.